src/main/resources/db/migration/common/V1_388__process_arm_rpo_pending_automated_task.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO darts.automated_task (aut_id, task_name, task_description, cron_expression, cron_editable, batch_size,
+                                  created_ts, created_by, last_modified_ts, last_modified_by)
+VALUES (nextval('aut_seq'), 'ProcessARMRPOPending', 'Update the record to STORED which has a status of ARM_RPO_PENDING and data_ingestion_ts is more than 24 hours',
```

Comment:
please dont use sequence, use hard coded values. This is so that id's are the same across environments.
We may need to do a tidy up of these numbers, and restart the sequence at something large, like 100.
Would you be able to add to the flyway to renumber all the tasks after 17 so they are all in order with no gaps?
![image](https://github.com/user-attachments/assets/c148410a-9500-4bd4-90cb-0f6b07db4717)


i have spoken to Hemanta, who says these numbers will be important in upcoming changes.

Reply:
Updated id to use a hard coded value + reset ids of all automated tasks